nav.go

```diff
@@ -14,6 +14,8 @@ import (
 	"strconv"
 	"strings"
 	"time"
+	"golang.org/x/text/collate"
+	"golang.org/x/text/language"
```

Comment:
Please group all the `github.com`/`golang.org` imports together in a section, similar to [`ui.go`](https://github.com/gokcehan/lf/blob/12e99fdb641565e3122ab62dce0b77e836aa69a4/ui.go#L17-L19). This should fix the CI build failure.

Reply:
Done.